benchmarks/ahi_hsd_benchmarks.py

```diff
@@ -74,18 +73,10 @@ def peakmem_save_true_color_to_geotiff(self):
         """Check peak memory usage of the generation and saving of true_color_nocorr."""
         self.save_true_color_nocorr_as_geotiff()
 
-    def load(self, composite):
-        """Load one composite."""
-        from satpy import Scene
-        scn = Scene(filenames=self.data_files, reader='ahi_hsd')
-        scn.load([composite], pad_data=False)
-        return scn
```

Comment:
I could be missing something but how can you remove this? All the other `compute_X` methods use `.load` still.

Reply:
This is now in the parent class

Reply:
But under a different name.

Reply:
`.load_no_padding` but `compute_B01` below still uses `.load`

Reply:
doh, indeed, thanks for spotting this!

Reply:
Code review!

Reply:
Fixed